api/main.go

```diff
@@ -25,52 +24,31 @@ func main() {
 		}
 	}
 
+	// Declare a new router with any middleware injected
 	r := middleware.NewRouter().Inject(handlers.LoggerHandler)
 	r.HandleFunc("/", handlers.RootHandler)
 
+	// Two-factor authentication
 	s := r.PathPrefix("/2fa").Subrouter()
 	s.HandleFunc("/{account}", handlers.TwofactorHandler)
 	s.HandleFunc("/{account}/verify", handlers.TwofactorVerifyHandler)
 	s.HandleFunc("/{account}/email", handlers.TwofactorEmailHandler)
 	s.HandleFunc("/{account}/reset", handlers.TwofactorResetHandler)
 
+	// Authentication schemes
 	o := r.PathPrefix("/auth").Subrouter()
 	o.HandleFunc("/basic", handlers.BasicAuth).Methods("POST")
 	o.HandleFunc("/{service}", handlers.AuthServiceHandler)
 	o.HandleFunc("/{service}/callback", handlers.AuthCallbackHandler)
 
-	// Validation
-	v := r.PathPrefix("/validate").Subrouter()
-	v.HandleFunc("/ssn/{ssn}", handlers.ValidateSSN)
-	v.HandleFunc("/email", handlers.ValidateEmail)
+	// Account specific actions
+	a := r.PathPrefix("/me").Subrouter().Inject(handlers.JwtTokenValidatorHandler)
+	a.HandleFunc("/validate", handlers.Validate).Methods("POST")
+	a.HandleFunc("/save", handlers.Save).Methods("POST")
```

Comment:
Good question. We should actually listen on both (for at least `/save`). This should be compliant  to standards of HTTP 1.1 and 2.0.

Funny thing about 2.0 is they are talking about doing away with PUT and DELETE verbs.